### PR TITLE
Drawing tools

### PR DESCRIPTION
**Pull request by redweasel**
Closes: #967
Also the coordinate system drawing direction is now more natural.

### CODE REVIEW

File: src/control/tools/ArrowHandler.cpp [hunk 1]
```diff
@@ -57,7 +57,7 @@ void ArrowHandler::drawShape(Point& c, const PositionInputData& pos)
 
 		// set up the size of the arrowhead to be 1/8 the length of arrow
```
Comment (LittleHuba) [inline src/control/tools/ArrowHandler.cpp]: That comment is now missleading and should be changed to the new behaviour.

File: src/control/tools/CoordinateSystemHandler.cpp [hunk 2]
```diff
@@ -66,7 +66,7 @@ void CoordinateSystemHandler::drawShape(Point& c, const PositionInputData& pos)
 		if ( !this->modControl )	//draw out from starting point
 		{			
 			stroke->addPoint(p1);
```
Comment (LittleHuba) [inline src/control/tools/CoordinateSystemHandler.cpp]: This line has bad formatting. There seem to be many whitespaces at the end of the line that should not be there.

**Comment by LittleHuba** [comment]
Sorry but you have now fixed the formatting for the whole files which is not something we want. Please revert the last commit and only change the two lines I have highlighted. Otherwise it will be a huge pain to later find the commit that caused a bug when commits change lines only for formatting.

File: src/control/tools/CoordinateSystemHandler.cpp [hunk 3]
```diff
@@ -65,24 +65,15 @@ void CoordinateSystemHandler::drawShape(Point& c, const PositionInputData& pos)
 		
 		if ( !this->modControl )	//draw out from starting point
 		{
```
Comment (LittleHuba) [inline src/control/tools/CoordinateSystemHandler.cpp]: No luck now it is that line :wink:

**Comment by LittleHuba** [comment]
I just checked the other PRs and none conflict with your changes. So we can ignore formatting and do that in the Formatting PR that is still pending.
Once PR-Freeze is formally over this will be a lot easier!

Thanks for the work!

**Comment by redweasel** [comment]
@LittleHuba thanks for the help. I'm very new to GitHub and messed up a couple times here and learned a lot.

**Comment by LittleHuba** [comment]
No worries! Formatting is currently not really easy to get right as the whole codebase is not yet formatted correctly. That is what the PR-Freeze is about. But afterwards it will be one command to format the PR correctly.

Your actual work was very nicely done and I hope to see more work from you! We have a lot of issues which are suitable for beginners.

**Pull request merged**